gradio/mcp.py

```diff
@@ -157,6 +157,7 @@ async def handle_sse(request):
                         streams[1],
                         self.mcp_server.create_initialization_options(),
                     )
+                return Response()
```

Comment:
Why do we need this?  Is it possible to even reach this part of the code? The with block seems like it would block forever

Reply:
Yeah so I added it because I would see an exception being raised (None type is not callable or something like that) when a client would disconnect from the mcp server. So while it doesn't functionally change anything, it prevents this exception from being raised during disconnect